targets/targets.json

```diff
@@ -2460,7 +2460,7 @@
         "supported_toolchains": ["uARM", "ARM"],
         "inherits": ["Target"],
         "device_has": ["ANALOGIN", "I2C", "INTERRUPTIN", "PORTIN", "PORTINOUT", "PORTOUT", "PWMOUT", "RTC", "SERIAL", "SPI", "SPISLAVE", "STDIO_MESSAGES"],
-        "release_versions": ["2"]
+        "release_versions": ["2", "5"]
```

Comment:
No... We don't have time to support GCC_ARM and IAR.
It supports ARM compiler only for now.
We have May 1st contest coming.
Let me call you and others internally.